tests/test_meltano_utils.py

```diff
@@ -406,3 +406,71 @@ def test_sdk_about_parsing_skip_default_dates():
             "kind": "date_iso8601",
         }
     ]
+
+
+@pytest.mark.parametrize(
+    "input,expected",
+    [
+        # Official
+        [["meltanolabs", True, 0, "low"], "gold"],
+        [["meltanolabs", True, 0, "medium"], "gold"],
+        [["meltanolabs", True, 0, "high"], "gold"],
+        [["meltanolabs", True, 1, "low"], "gold"],
+        [["meltanolabs", True, 1, "medium"], "gold"],
+        [["meltanolabs", True, 1, "high"], "gold"],
+        [["meltanolabs", True, 6, "low"], "gold"],
+        [["meltanolabs", True, 6, "medium"], "gold"],
+        [["meltanolabs", True, 6, "high"], "gold"],
+        [["meltanolabs", False, 0, "low"], "gold"],
+        [["meltanolabs", False, 0, "medium"], "gold"],
+        [["meltanolabs", False, 0, "high"], "gold"],
+        [["meltanolabs", False, 1, "low"], "gold"],
+        [["meltanolabs", False, 1, "medium"], "gold"],
+        [["meltanolabs", False, 1, "high"], "gold"],
+        [["meltanolabs", False, 6, "low"], "gold"],
+        [["meltanolabs", False, 6, "medium"], "gold"],
+        [["meltanolabs", False, 6, "high"], "gold"],
+
+        # Partner
+        [["matatika", True, 0, "low"], "gold"],
+        [["matatika", True, 0, "medium"], "gold"],
+        [["matatika", True, 0, "high"], "gold"],
+        [["matatika", True, 1, "low"], "gold"],
+        [["matatika", True, 1, "medium"], "gold"],
+        [["matatika", True, 1, "high"], "gold"],
+        [["matatika", True, 6, "low"], "gold"],
+        [["matatika", True, 6, "medium"], "gold"],
+        [["matatika", True, 6, "high"], "gold"],
+        [["matatika", False, 0, "low"], "bronze"],
+        [["matatika", False, 0, "medium"], "bronze"],
+        [["matatika", False, 0, "high"], "bronze"],
+        [["matatika", False, 1, "low"], "bronze"],
+        [["matatika", False, 1, "medium"], "silver"],
+        [["matatika", False, 1, "high"], "silver"],
+        [["matatika", False, 6, "low"], "bronze"],
+        [["matatika", False, 6, "medium"], "silver"],
+        [["matatika", False, 6, "high"], "silver"],
+
+        # Community
+        [["foobar", True, 0, "low"], "silver"],
+        [["foobar", True, 0, "medium"], "silver"],
+        [["foobar", True, 0, "high"], "silver"],
+        [["foobar", True, 1, "low"], "silver"],
+        [["foobar", True, 1, "medium"], "silver"],
+        [["foobar", True, 1, "high"], "silver"],
+        [["foobar", True, 6, "low"], "silver"],
+        [["foobar", True, 6, "medium"], "silver"],
+        [["foobar", True, 6, "high"], "silver"],
```

Comment:
I would think these could be Gold. Just b/c it's a community connector doesn't mean it _can't_ be Gold. It's just not automatic.

Reply:
@tayloramurphy ah yes youre right.